SourceX/DiabloUI/button.cpp

```diff
@@ -8,18 +8,16 @@ Art SmlButton;
 
 void LoadSmlButtonArt()

```

Comment:
For some reason the link is unable to find this function when moved to dialog.cpp?

Reply:
What's the error message?

Reply:
This is an ODR violation, the function should be removed from here and `button.h`

Reply:
```
Undefined symbols for architecture x86_64:
  "dvl::LoadSmlButtonArt()", referenced from:
      dvl::progress_Load(char*) in progress.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [devilutionx.app/Contents/MacOS/devilutionx] Error 1
make[1]: *** [CMakeFiles/devilutionx.dir/all] Error 2
make: *** [all] Error 2

```

Reply:
This is because in `dialog.cpp` it's defined in an anonymous namespace.

Why move it btw? Is it only ever used in dialogs? Never in bnet UI?

Reply:
Arh had missed that, thanks. Yeah, your right this shouldn't be moved I think I just moved it as I worked on it and then didn't take a step back to look at the overall thing. This whole PR grew a lot bigger then I had expected initially.